Translation French

armeirlaen opened the pull request:
Translate the app in french

# Description

This pull request adds French translation to the application. 

Fixes # (issue)

## Type of change

Please delete options that are not relevant.

- [ ] New feature (non-breaking change which adds functionality)

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- Runned the app

**Test Configuration**:
* Firmware version:
* Hardware:
* Toolchain:
* SDK:

# Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

Kubis10 commented on langs/Francais.py:
```diff
@@ -0,0 +1,34 @@
+TITLE_NUMBER_OF_CHATS = 'Nombre de conversations'
+TITLE_SENT_MESSAGES = 'Messages envoyés'
+TITLE_TOTAL_MESSAGES = 'Tous les messages'
+TITLE_CLOSE_POPUP = 'Fermer'
```
Hi! Thanks for committing to my project and translating it but you miss one line: TITLE_TOTAL_CHARS = 'Total Characters'  as you can see in English translation.

armeirlaen replied:
Yes sorry i must have deleted it by mistake. I will add it

armeirlaen commented:
I also added a new feature to see and sort photos. There was an issue in the sense that Total call duration was not displaying Total call duration but Total char count. I fixed that allong with adding a new column for Photos. I've not adapted the translation for this new column but can do so if necessary. Sorry to do that in the same pull request than the translation i don't know how to open a different pull request from the same branch

Kubis10 commented:
To open other pr you would need to create new branch in your fork, but thanks again for this pr
